packages/server/app/Controllers/Http/MyProfileController.test.ts

```diff
@@ -34,27 +60,24 @@ test.group('Users Profile information tests', (group) => {
         await Database.rollbackGlobalTransaction();
     });
 
-    test('It should retrieve my profile information', async (assert) => {
-        const userID = datatype.uuid();
-        const userNickname = internet.userName();
-        await createUserAndGetSocket({
-            userNickname,
-            userID,
+    test('Retrieves profile information of the current authenticated user', async (assert) => {
+        const request = supertest.agent(BASE_URL);
```

Comment:
**👏 praise:** really nice, it also works if not iterating on already existing seeded user